src/cdk-experimental/combobox/combobox-popup.ts

```diff
@@ -33,11 +41,21 @@ export class CdkComboboxPopup<T = unknown> implements OnInit {
   }
   private _role: AriaHasPopupValue = 'dialog';
 
+  @Input()
+  get firstFocus(): HTMLElement {
+    return this._firstFocusElement;
+  }
+  set firstFocus(id: HTMLElement) {
+    this._firstFocusElement = id;
+  }
+  private _firstFocusElement: HTMLElement;
```

Comment:
I don't quite follow this; if this directive is for dialog combobox content, I think the focus element would always be `this.elementRef.nativeElement`

Reply:
My thought process was that if the popup is a dialog, then this.elementRef.nativeElement.focus() will just focus the entire popup. With this input, the user could choose to immediately focus some interior element like an input or listbox inside the dialog.

Reply:
We should be able to take care of that with the focus trap from cdk/a11y without having to add direct support for it to the pop-up here (in fact `CdkDialog` would take care of that automatically).